indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -101,30 +104,42 @@ FileType.defaultProps = {
   uploads: {},
 };
 
-export default function FileManager({uploadURL, fileTypes, files}) {
+export default function FileManager({downloadURL, uploadURL, fileTypes, files}) {
   const _fileTypes = useMemo(() => mapFileTypes(fileTypes, files), [fileTypes, files]);
   const [state, dispatch] = useReducer(reducer, {
     fileTypes: _fileTypes,
     uploads: {},
   });
 
   return (
-    <div styleName="file-manager">
-      <FileManagerContext.Provider value={dispatch}>
-        {state.fileTypes.map(fileType => (
-          <FileType
-            key={fileType.id}
-            uploadURL={uploadURL}
-            fileType={fileType}
-            uploads={state.uploads[fileType.id]}
-          />
-        ))}
-      </FileManagerContext.Provider>
+    <div styleName="file-manager-wrapper">
+      <div styleName="file-manager">
+        <FileManagerContext.Provider value={dispatch}>
+          {state.fileTypes.map(fileType => (
+            <FileType
+              key={fileType.id}
+              uploadURL={uploadURL}
+              fileType={fileType}
+              uploads={state.uploads[fileType.id]}
+            />
+          ))}
+        </FileManagerContext.Provider>
+      </div>
+      {downloadURL && files.length !== 0 && (
+        <div>
+          <Button as="a" href={downloadURL} floated="right" icon primary>
+            <Translate>
+              <Param name="icon" value={<Icon name="download" />} /> Download ZIP
```

Comment:
Just move the icon outside the Translate..

Reply:
So, I did it like that because prettier was complaining when I added `{' '}` after `<Icon />` componentrr

Reply:
how did it complain (and without being able to fix it)?

Reply:
It complained about `{' '}`. The way to fix it was to remove it which made the icon inside the button look weird. That is a bit weird since we have a few other places where we do sth similar. Maybe I will take a closer look at that.